pallets/asset-registry/src/types.rs

```diff
@@ -26,16 +26,15 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Encode, Decode, Eq, PartialEq, Copy, Clone, RuntimeDebug, TypeInfo, MaxEncodedLen)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub enum AssetType<AssetId> {
+pub enum AssetType {
 	Token,
-	PoolShare(AssetId, AssetId), // Use XYX instead
```

Comment:
this would require migration in basislisk ?

Reply:
yes